services/net-service.ts

```diff
@@ -71,6 +77,53 @@ export class Net implements INet {
 
 		return startPort;
 	}
+
+	public async waitForPortToListen(port: number, timeout: number, interval?: number): Promise<boolean> {
+		interval = interval || Net.DEFAULT_INTERVAL;
+		const endTime = new Date().getTime() + timeout;
+		const platformData: IDictionary<any> = {
```

Comment:
You can introduce an strongly typed interface here instead of `any`. Not a merge-stopper though

Reply:
No need to introduce an interface for an object that's created and used inside a single method.